### ISSUE TITLE
[Question] Multiple Pixel IDs

### ISSUE THREAD

**Issue by skube** [zsajjad/react-facebook-pixel#14]
Is it possible to implement more than one Pixel per site?

**Comment by aejay-rs** [comment]
Stumbled across this question and it happened to be something that came up before when we were researching pixel implementations for our site. I hope you aren't still waiting for an answer, but in case there are others that have this concern:

From what I understand, you can initialize the Facebook pixel multiple times, once for each pixel ID, and follow up calls for tracking will submit events for each pixel ID. They have mentioned this in some blog posts, but the multiple trackers behavior isn't covered well in their documentation. In theory, you should be able to do the same with this library for that behavior:

```js
ReactPixel.init('FIRST_PIXEL_ID');
ReactPixel.init('SECOND_PIXEL_ID');
```

However, if you want to use the "trackSingle" or "trackSingleCustom" methods to track events to a particular pixel ID tracker, that behavior is not implemented by this library, so you'll have to do it yourself.

You should be able to do that by hitting the fbq object exposed by the type:

```js
ReactPixel.fbq('trackSingle', 'FIRST_PIXEL_ID', 'PageView');
```

Hope this helps. We'll be doing an implementation around these lines soon, so if I run into any snags with this I will be sure to mention those here.

**Comment by mattgenious** [comment]
https://github.com/zsajjad/react-facebook-pixel/pull/13
is a pull request from https://github.com/jasondecastro with the functionality asked for

**Comment by zsajjad** [comment]
Merged #24
